### PR TITLE
Home header

### PR DESCRIPTION
**Pull request by basnetlaxmi**
## Home Header 
In this version : 
⚪ I added h1 text in index.html.
⚪ I added style for h1 of index.html with style.css.

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# This file should be ignored by git.
```
Comment (AshakaE) [inline test.md]: - [ ] Please make sure this file is properly ignored from git before making a `git push` 🙏

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @basnetlaxmi  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!

#  Required Changes ♻️ 🚧
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,77 @@
-# hello-microverse
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# hello-microverse
+
+> This is a site showing hello microverse text as a header.
+
+![screenshot](./images/pic.png)
```
Comment (DanteAlonsoHT) [inline README.md]: This image is not working, you should remove it or fix it as you want.

- [ ] Remove or Fix the image in your README file.

File: README.md [hunk 3]
```diff
@@ -1 +1,77 @@
...
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```
Comment (DanteAlonsoHT) [inline README.md]: Remove "Setup", "Install", "Usage", "Run tests" and "Deployment" sections if you are not using them.

- [ ] Remove empty sections.

File: README.md [hunk 4]
```diff
@@ -1 +1,77 @@
-# hello-microverse
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# hello-microverse
+
+> This is a site showing hello microverse text as a header.
```
Comment (DanteAlonsoHT) [inline README.md]: I know you did a good effort creating your description for your project, but I strongly recommend you adding a much better description, in you want a reference, 3 or more lines can be good, you can explain what you did, what kind of technologies you used, what is the scope, and so on.

- [ ] Improve your description for your project.

File: README.md [hunk 5]
```diff
@@ -1 +1,77 @@
...
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (DanteAlonsoHT) [inline README.md]: You should leave only one author in this solo project, and please add the links for your own social media.

- [ ] Remove extra author information.
- [ ] Add your own social media links in this section of your README file.

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1>Status: Changes Required 🚫 ♻️  </h1>

Hi @basnetlaxmi, 

Nice work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet, please follow the next changes required in order to improve your project.

 ## Required Changes ♻️

_Check the comments under the review._

<hr>

I know you have the capabilities enough to finish these changes required in time 😉 

Happy coding!👏👏 😉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by dasileker — APPROVED** [review]
### **Hi @basnetlaxmi**

### **Project Approved ✅** 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**